src/components/map/map-component.tsx

```diff
@@ -318,4 +326,23 @@ export class MapComponent extends BaseComponent<IProps, IState>{
       }
     }
   }
+
+  private getRiskItems = () => {
+    // TODO: this code adds a risk map item for every sample collection with threshold kTephraThreshold
+    // need to specify correct samples to add risk item and correct threshold
+    const { samplesCollectionsStore } = this.stores;
+    const riskItems: any[] = [];
+    samplesCollectionsStore.samplesCollections.forEach( (samplesCollection: any, i) => {
+      const thresholdData: ThresholdData = calculateThresholdData(samplesCollection.samples, kTephraThreshold);
+      const riskLevel = calculateRisk(thresholdData.greaterThanPercent);
+      riskLevel && riskItems.push(
+        <Marker
+          position={[samplesCollection.x, samplesCollection.y]}
+          icon={riskIcon(riskLevel.iconColor, riskLevel.iconText, true)}
+          key={"risk-" + i}
+        />
+      );
+    });
+    return riskItems;
+  }
```

Comment:
This function is where we will need to determine which samplesCollection actually have risk level icon shown on the map and which threshold to actually use (kTephraThreshold is just for testing).